src/package.js

```diff
@@ -1156,16 +1158,22 @@ module.exports = class Package {
           '..',
           '..'
         );
-        nativeModulePaths.push(nativeModulePath);
+        const nodeFilePaths = nativeModulePaths.get(nativeModulePath) || [];
+        nativeModulePaths.set(
+          nativeModulePath,
+          nodeFilePaths.push(relativeNativeModuleBindingPath)
+        );
```

Comment:
We need to be able to get the `.node` file path, but I cannot understand how this part of the code works. Atom never gets in this path when I run it in `--dev` mode. So, not sure if this part is correct.

Reply:
OK, there is an example of such a package in the tests.

```json
{
  "name": "package-with-ignored-incompatible-native-module",
  "version": "1.0.0",
  "main": "./main.js",
  "_atomModuleCache": {
    "extensions": {
      ".node": [
        "node_modules/compatible-native-module/build/Release/native.node"
      ]
    }
  }
}

```

Reply:
Fixed